src/test/java/converter/UserToStringConverterTest.java

```diff
@@ -0,0 +1,29 @@
+package converter;
+
+import model.User;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+
+public class UserToStringConverterTest {
+
+    final UserToStringConverter userToStringConverter = new UserToStringConverter();
+
+    /*  @Test
+  public void createdUser() {
+User user = new User("igor","igor_password","ihar","osipov");
+User newUser = Mockito.mock(User.class);
+assertEquals("ihar",newUser.getSurname());
+   }
+*/
+    @Test
+    public void testConvert() {
```

Comment:
Add test for converting null object and user with null fields

Reply:
corrected

Reply:
see only for null user
what string will be written in file in case `User(null, null, null, null)` ?